Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
+    :param level: log level of the message
+    :type level: ``pyDKB.common.types.logLevel`` member
+    :param *args: additional prefixes (will be output between log
+                  level prefix and message body)
+    :type *args: str
+    """
+    if not logLevel.hasMember(level):
+        log("Unknown log level: %s" % level, logLevel.WARN)
+        level = logLevel.INFO
+    if type(message) == list:
+        lines = message
+    else:
+        lines = message.splitlines()
+    if args:
+        prefix = ' ' + ' '.join(['(%s)' % p for p in args])
+    else:
+        frm = inspect.stack()[1]
+        mod = inspect.getmodule(frm[0])
+        modname = getattr(mod, '__name__', 'main')
+        prefix = ' (%s)' % modname
+    if lines:
+        dtime = datetime.now().strftime(DTFORMAT)
+        out_message = "%s (%s)%s %s" % (dtime, logLevel.memberName(level),
+                                        prefix, lines[0])
+        for l in lines[1:]:
+            out_message += "\n(==) %s" % l
```

Comment:
Similar to comment above - maybe adjust the lines' alignment?

```
2019-08-26 15:07:38 (INFO) (main) 1
(==) 2
(==) 3
(==) 4
(==) 5
```
to
```
2019-08-26 15:07:38 (INFO) (main) 1
                             (==) 2
                             (==) 3
                             (==) 4
                             (==) 5
```
?

Reply:
Toooooo wide empty space: with `(main)` you have 30 spaces; with `(ProcessorStage)` it\`s gonna be even more. Since logs are more likely to be checked in terminal, 80-characters width is still worth remembering ;) Imagine longer lines:
```
2019-08-26 15:07:38 (INFO) (main) Some error occured: 
                             (==) here are some details about the error: and HERE IS THE MOST IMPORTANT INFORMATION
                             (==) another piece of details: and YET ANOTHER PIECE OF IMPORTANT INFORMATION
```

Reply:
Fair enough.